server/data_processors/DashboardDataProcessor/index.js

```diff
@@ -0,0 +1,62 @@
+import _ from 'lodash'
+import CollectionsModel from '../../models/CollectionsModel'
+
+class DashboardDataProcessor {
+  #configDataMap = new Map()
+  #assessmentsFromConfig
+
+  constructor(assessmentsFromConfig, configuration, db) {
+    this.#assessmentsFromConfig = assessmentsFromConfig()
```

Comment:
instead of passing this in as a function that we execute in the constructor, why not pass the resulting data from executing this function?

Reply:
[fba1285](https://github.com/AMP-SCZ/dpdash/pull/344/commits/fba128510aee9607c64d5855d8d8fcad4e94155f)